tests/test_patch.py

```diff
@@ -0,0 +1,51 @@
+import contextlib
+import tempfile
+from functools import wraps
+from multiprocessing import Process
+
+
+def run_test_in_subprocess(func):
```

Comment:
This file contains the relevant changes. See commit [60b8d7a](https://github.com/rapidsai/rapids-dask-dependency/pull/37/commits/60b8d7a04d1ef71979d727405651d6eda60bfa6f).